examples/with-linaria/app/page.tsx

```diff
@@ -0,0 +1,190 @@
+import Image from 'next/image';
+import { styled } from '@linaria/react';
+
+const Main = styled.main`
+  min-height: 100vh;
+  padding: 4rem 0;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const Title = styled.h1`
+  margin: 0;
+  line-height: 1.15;
+  font-size: 4rem;
+  font-style: normal;
+  font-weight: 800;
+  letter-spacing: -0.025em;
+
+  & a {
+    text-decoration: none;
+    color: #0070f3;
+  }
+
+  & a:hover,
+  & a:focus,
+  & a:active {
+    text-decoration: underline;
+  }
+  & h2 {
+    margin: 0 0 1rem 0;
+    font-size: 1.5rem;
```

Comment:
```css
  & h2 { ... }
  & p { ... }
```   
is probably supposed to be part of `Card`, not `Title`, since `Title` has no `<h2>` or `<p>` inside

Reply:
@none23 good catch, fixed it!